### PR TITLE
13.0 l10n_il retention report

### PR DESCRIPTION
**Pull request by nla-odoo**
Task: https://www.odoo.com/web#id=1966458&action=327&model=project.task&view_type=form&cids=2&menu_id=4720

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_il/models/l10n_il_ita_branch.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+from odoo import fields, models
+
+
+class ITABranch(models.Model):
+    _name = 'l10n.il.ita.branch'
+
+    name = fields.Char(string='ITA Branch')
+    code = fields.Char(string='ITA Branch Code')
```
Comment (jco-odoo) [inline addons/l10n_il/models/l10n_il_ita_branch.py]: 1st of all, we can not change the schema in v13 for l10n_il.   I would say, again with a selection, but it is possible we need the name also in Hebrew, which is not reflected here...

File: addons/l10n_il/models/l10n_il_tax_reason.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+from odoo import fields, models
+
+
+class WithhReason(models.Model):
+    _name = 'l10n.il.tax.reason'
+
+    name = fields.Char(string='Withh Tax Reason')
+    code = fields.Char(string='Code for Withh Tax Reason')
+
+    _sql_constraints = [('code_name_uniq', 'unique (code,name)', 'The code of the Withholding Tax Reason must be unique!')]
```
Comment (jco-odoo) [inline addons/l10n_il/models/l10n_il_tax_reason.py]: selection field

File: addons/l10n_il/models/account_tax.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+
+from odoo import fields, models
+
+
+class WithhTaxReportData(models.Model):
+    _inherit = 'account.tax'
+
+    l10n_il_tax_reason = fields.Many2one('l10n.il.tax.reason', string='Tax Reason', help="This field contains the withholding tax reason that will be used for Annual Witholding Tax Report'")
```
Comment (jco-odoo) [inline addons/l10n_il/models/account_tax.py]: I think this needs to be on the partner as well.  (in the report, we group by partner...)

**Review by jco-odoo — COMMENTED** [review]
Check new spec and this needs to follow stable policy, so new fields need to go to the l10n_il_reports module.

**Comment by lathama** [comment]
@nla-odoo knowing that 13 is no longer updated, is this PR still needed?

**Comment by C3POdoo** [comment]
Dear @nla-odoo,

Thank you for your contribution but the version 13.0 is no longer supported.
We only support the last 3 stable versions so no longer accepts patches into this branch.

We apology if we could not look at your request in time.
If the contribution still makes sense for the upper version, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

_This is an automated message._

**Pull request closed without merging**